src/providers/TokensProvider.tsx

```diff
@@ -77,6 +86,7 @@ export default function TokensProvider({
       if (!tokenInfos) return
       Object.keys(tokenInfos).forEach((address) => {
         ;(tokenInfos[address] as BasicToken).isLPToken = lpTokens.has(address)
+        ;(tokenInfos[address] as BasicToken).typeAsset = tokenType[address]
```

Comment:
I think it would be good to use a fake price for `OTHER` tokens on localhost, just so we can still see APRs and things like that. Can be literally any price.

I think I would prefer to default type to `OTHER` just because dealing with optional types is annoying.

Reply:
Oh, I got you!